modules/framework/src-js/PlatformTask.scala

```diff
@@ -0,0 +1,19 @@
+package weaver
+
+import sbt.testing.{ EventHandler, Logger, Task }
+
+private[weaver] trait PlatformTask extends AsyncTask {
+
+  override def execute(
+      eventHandler: EventHandler,
+      loggers: Array[Logger],
+      continuation: Array[Task] => Unit): Unit = {
+    val _ = executeFuture(eventHandler, loggers).map(_ =>
+      continuation(Array.empty[Task]))(
+      scala.scalajs.concurrent.JSExecutionContext.queue)
```

Comment:
Hmm, is this what we had before?
IIRC we are bringing the macrotask - do we want to use it instead? https://github.com/disneystreaming/weaver-test/issues/476

Reply:
In this case it really doesn't matter

Reply:
> Hmm, is this what we had before?

Nah before we were using a callback based async method. We're unifying between JS and Native by relying on Future instead, and we're only needing an EC to plug the "end of the world" callback to the future.